src/routes/docs/products/functions/execute/+page.markdoc

```diff
@@ -12,34 +12,35 @@ Executions can be invoked through the Appwrite SDK and visiting its REST endpoin
 Here are all the different ways to consume your Appwrite Functions.
 
 # Domains {% #domains %}
-
+You can execute a function through HTTP requests, using a browser or by sending an HTTP request.
 1. In the Appwrite Console's sidebar, click **Functions**.
-1. Under the **Domains** tab, you'll find the generated domain from Appwrite and your custom domains.
+1. Under **Execute access**, set the access to `Any` so that anyone can execute the function. You will use [JWTs](/docs/products/auth/jwt) to authenticate users.
+1. Under the **Domains** tab, you'll find the generated domain from Appwrite and your custom domains. [Learn about adding a custom domain](/docs/products/functions/domains).
```

Comment:
These steps are both 1.

Reply:
Yes markdown renders them as 1.2.3.4 ;) This in fact works